ui/client/containers/Processes.js

```diff
@@ -102,67 +102,68 @@ class Processes extends PeriodicallyReloadingComponent {
             </span>
           </div>
-          {this.props.loggedUser.isWriter ? (
-          <div id="process-add-button" className="big-blue-button input-group " role="button"
-               onClick={() => this.setState({showAddProcess : true})}>CREATE NEW PROCESS
-                             <img id="add-icon" src={createProcessIcon} />
-                           </div>) : null
+          {
+            this.props.loggedUser.isWriter ? (
+              <div id="process-add-button"
+                   className="big-blue-button input-group "
+                   role="button"
+                   onClick={() => this.setState({showAddProcess : true})}
+              >
+                CREATE NEW PROCESS
+                <img id="add-icon" src={createProcessIcon} />
+              </div>
+            ) : null
           }
         </div>
+
         <AddProcessDialog onClose={() => this.setState({showAddProcess : false})} isOpen={this.state.showAddProcess} isSubprocess={false}/>
         <LoaderSpinner show={this.state.showLoader}/>
         <Table className="esp-table"
                onSort={sort => this.setState({sort: sort})}
                onPageChange={currentPage => this.setState({currentPage: currentPage})}
-           noDataText="No matching records found."
-           hidden={this.state.showLoader}
-           currentPage={this.state.currentPage}
-           defaultSort={this.state.sort}
-           itemsPerPage={10}
-           pageButtonLimit={5}
-           previousPageLabel="<"
-           nextPageLabel=">"
-           sortable={['name', 'category', 'modifyDate']}
-           filterable={['name', 'category']}
-           hideFilterInput
-           filterBy={this.getFilterValue()}
-
+               noDataText="No matching records found."
+               hidden={this.state.showLoader}
+               currentPage={this.state.currentPage}
+               defaultSort={this.state.sort}
+               itemsPerPage={10}
+               pageButtonLimit={5}
+               previousPageLabel="<"
+               nextPageLabel=">"
+               sortable={['Process name', 'Category', 'Last modification']}
+               filterable={['Process name', 'Category']}
+               hideFilterInput
+               filterBy={this.getFilterValue()}
         >
 
-          <Thead>
-            <Th column="name">Process name</Th>
-            <Th column="category">Category</Th>
-            <Th column="modifyDate" className="date-column">Last modification</Th>
-            <Th column="status" className="status-column">Status</Th>
-            <Th column="edit" className="edit-column">Edit</Th>
-            <Th column="metrics" className="metrics-column">Metrics</Th>
-          </Thead>
-
           {this.state.processes.map((process, index) => {
             return (
               <Tr className="row-hover" key={index}>
-                <Td column="name" value={process.name}>
+                <Td column="Process name" value={process.name}>
```

Comment:
Let's pass column labels as 
columns =       {key: 'name', label: 'Name'},
      {key: 'category', label: 'Action'}, ...
    ]
and then <Table columns={columns}>

This still works, and we avoid passing labels as keys, which is kind of weird ;)

Reply:
I was looking for this option ... :) Thanks!